[Bug] `/list` command fails if a backend server cannot be pinged

In my Velocity config, I have a test server defined that I'm not always running.
If the `/list` command fails to ping a server, it dies on an exception.
```
[12:16:09] [JDA MainWS-ReadThread/ERROR]: One of the EventListeners had an uncaught exception
java.lang.RuntimeException: java.util.concurrent.ExecutionException: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:44444
	at ooo.foooooooooooo.velocitydiscord.discord.commands.ListCommand.handle(ListCommand.java:36) ~[?:?]
	at ooo.foooooooooooo.velocitydiscord.discord.Discord.onSlashCommandInteraction(Discord.java:187) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.api.hooks.ListenerAdapter.onEvent(ListenerAdapter.java:382) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.api.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:96) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.hooks.EventManagerProxy.handleInternally(EventManagerProxy.java:88) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.hooks.EventManagerProxy.handle(EventManagerProxy.java:70) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.JDAImpl.handleEvent(JDAImpl.java:170) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.handle.InteractionCreateHandler.handleCommand(InteractionCreateHandler.java:109) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.handle.InteractionCreateHandler.handleInternally(InteractionCreateHandler.java:80) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:39) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:953) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:840) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:818) ~[?:?]
	at velocitydiscord.net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:992) ~[?:?]
	at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385) ~[?:?]
	at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276) ~[?:?]
	at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996) ~[?:?]
	at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755) ~[?:?]
	at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108) ~[?:?]
	at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64) ~[?:?]
	at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45) ~[?:?]
Caused by: java.util.concurrent.ExecutionException: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:44444
	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at ooo.foooooooooooo.velocitydiscord.discord.commands.ListCommand.handle(ListCommand.java:33) ~[?:?]
	... 20 more
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: /127.0.0.1:44444
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
	at io.netty.channel.unix.Errors.newConnectException0(Errors.java:155) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.channel.unix.Errors.handleConnectErrno(Errors.java:128) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.channel.unix.Socket.finishConnect(Socket.java:321) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.doFinishConnect(AbstractEpollChannel.java:710) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.finishConnect(AbstractEpollChannel.java:687) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.epollOutReady(AbstractEpollChannel.java:567) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.channel.epoll.EpollEventLoop.processReady(EpollEventLoop.java:477) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:385) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[velocity-3.1.2-SNAPSHOT-175.jar:3.1.2-SNAPSHOT (git-4b33d5fb-b175)]
	at java.lang.Thread.run(Thread.java:833) [?:?]
```
I propose that servers defined in Velocity's config that cannot be pinged be present in the `/list` command with something like "Server cannot be reached" in place of the player list.

--- a/README.md
+++ b/README.md
@@ -67,6 +67,7 @@ ephemeral = true
 server_format = "[{server_name} {online_players}/{max_players}]"
 player_format = "- {username}"
 no_players = "No players online"
+server_offline = "Server offline"
 codeblock_lang = "asciidoc"
 
 # Discord > Minecraft message formats

--- a/src/main/java/ooo/foooooooooooo/velocitydiscord/Config.java
+++ b/src/main/java/ooo/foooooooooooo/velocitydiscord/Config.java
@@ -54,6 +54,7 @@ public class Config {
     public String DISCORD_LIST_SERVER_FORMAT = "[{server_name} {online_players}/{max_players}]";
     public String DISCORD_LIST_PLAYER_FORMAT = "- {username}";
     public String DISCORD_LIST_NO_PLAYERS = "No players online";
+    public String DISCORD_LIST_SERVER_OFFLINE = "Server offline";
     public String DISCORD_LIST_CODEBLOCK_LANG = "asciidoc";
 
     // minecraft formats
@@ -141,6 +142,7 @@ public class Config {
         DISCORD_LIST_SERVER_FORMAT = toml.getString("discord.commands.list.server_format", DISCORD_LIST_SERVER_FORMAT);
         DISCORD_LIST_PLAYER_FORMAT = toml.getString("discord.commands.list.player_format", DISCORD_LIST_PLAYER_FORMAT);
         DISCORD_LIST_NO_PLAYERS = toml.getString("discord.commands.list.no_players", DISCORD_LIST_NO_PLAYERS);
+        DISCORD_LIST_SERVER_OFFLINE = toml.getString("discord.commands.list.server_offline", DISCORD_LIST_SERVER_OFFLINE);
         DISCORD_LIST_CODEBLOCK_LANG = toml.getString("discord.commands.list.codeblock_lang", DISCORD_LIST_CODEBLOCK_LANG);
 
         DISCORD_CHUNK = toml.getString("minecraft.discord_chunk", DISCORD_CHUNK);

--- a/src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java
+++ b/src/main/java/ooo/foooooooooooo/velocitydiscord/discord/Discord.java
@@ -52,7 +52,7 @@ public class Discord extends ListenerAdapter {
         this.logger = logger;
         this.config = config;
 
-        commands.put("list", new ListCommand(server, config));
+        commands.put("list", new ListCommand(server, logger, config));
 
         var messageListener = new MessageListener(server, logger, config);
 

--- a/src/main/java/ooo/foooooooooooo/velocitydiscord/discord/commands/ListCommand.java
+++ b/src/main/java/ooo/foooooooooooo/velocitydiscord/discord/commands/ListCommand.java
@@ -6,13 +6,16 @@ import ooo.foooooooooooo.velocitydiscord.Config;
 import ooo.foooooooooooo.velocitydiscord.util.StringTemplate;
 
 import java.util.concurrent.ExecutionException;
+import java.util.logging.Logger;
 
 public class ListCommand implements ICommand {
     private final ProxyServer server;
+    private final Logger logger;
     private final Config config;
 
-    public ListCommand(ProxyServer server, Config config) {
+    public ListCommand(ProxyServer server, Logger logger, Config config) {
         this.server = server;
+        this.logger = logger;
         this.config = config;
     }
 
@@ -32,7 +35,9 @@ public class ListCommand implements ICommand {
             try {
                 var b = server.ping().get().asBuilder();
                 maxPlayers = b.getMaximumPlayers();
-            } catch (ExecutionException | InterruptedException e) {
+            } catch (ExecutionException e) {
+                logger.warning("Could not ping server: " + e);
+            } catch (InterruptedException e) {
                 throw new RuntimeException(e);
             }
 
@@ -45,7 +50,11 @@ public class ListCommand implements ICommand {
                     .toString()
             ).append("\n");
 
-            if (playerCount == 0) {
+            if (maxPlayers == 0) {
+                if (!config.DISCORD_LIST_SERVER_OFFLINE.isEmpty()) {
+                    sb.append(config.DISCORD_LIST_SERVER_OFFLINE).append("\n");
+                }
+            } else if (playerCount == 0) {
                 if (!config.DISCORD_LIST_NO_PLAYERS.isEmpty()) {
                     sb.append(config.DISCORD_LIST_NO_PLAYERS).append("\n");
                 }

--- a/src/main/resources/config.toml
+++ b/src/main/resources/config.toml
@@ -60,6 +60,7 @@ ephemeral = true
 server_format = "[{server_name} {online_players}/{max_players}]"
 player_format = "- {username}"
 no_players = "No players online"
+server_offline = "Server offline"
 codeblock_lang = "asciidoc"
 
 # Discord > Minecraft message formats

